### DISCUSSION TITLE
Moving issue number to the front of the <title> tag would make it easier to reference issue numbers

### DISCUSSION THREAD

**Discussion by markjaquith** [community/community#63337] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

I have a recurring "papercut" in my daily use of GitHub issues. Here's the scenario:

I open a GitHub issue to work on a bug. While researching it, I open a few more browser tabs.

Eventually I fix the issue, and go to make the commit. I type my commit message and then "fixes #"... and then I look back at my browser to see what the issue number was.

But I see this:

<img width="222" alt="CleanShot 2023-08-09 at 19 41 25@2x" src="https://github.com/community/community/assets/353790/6338a9b5-6446-4919-a088-bb7d358d7d6d">

That's my issue... but the issue number is buried deep within its `<title>` tag. So than I have to go over there, click on the tab, find the issue number in the URL or on the page, and then go back and type it in.

Starting the `<title>` tag with the issue name is not super useful to me. I would find this massively more useful:

<img width="224" alt="CleanShot 2023-08-09 at 19 42 43@2x" src="https://github.com/community/community/assets/353790/972848d0-e373-4e14-ae43-04914d2adca4">

I can still see the start of the title to confirm it's the right issue, but importantly, the all-powerful issue number is front-and-center.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by colorful-tones** [comment]
OMG Yes!!!
